src/components/form/input/input.vue

```diff
@@ -1,16 +1,16 @@
 <template>
     <div class="form-field form-field__input" :class="rootClasses">
         <label>
-            <span class="form-field--title-label" v-if="!hasFloatingLabel && $slots.label">
+            <span class="form-field--title-label" v-if="!hasFloatingLabel && $scopedSlots.label">
```

Comment:
Are you sure that `$scopedSlots` will work also on old vue@2 versions?

Reply:
I'm sure that it will break. But that's sort of the point. We'll enforce `vue@^2.6.0` with the major version bump. I've added a peer dependency that should throw a warning.